### PR TITLE
Check pep8

### LINKED ISSUE #19: PEP8 should be compulsory on travis
Travis must check if PEP8 is ok.

You can copy/paste this file : 
https://github.com/inasafe/inasafe/blob/develop/safe/test/test_pep8.py

### PR DESCRIPTION
**Pull request by dimasciput**
Fix #19

### CODE REVIEW

File: cadasta/test/test_init.py [hunk 1]
```diff
@@ -55,8 +55,11 @@ def test_read_init(self):
         metadata.extend(parser.items('general'))
 
         for expectation in required_metadata:
-            message = ('Cannot find metadata "%s" in metadata cadasta (%s).' % (
-                expectation, file_path))
+            message = (
+                'Cannot find metadata "%s" '
```
Comment (Gustry) [inline cadasta/test/test_init.py:59]: You should use translate function when you use strings.

File: cadasta/gui/tools/cadasta_login.py [hunk 2]
```diff
@@ -60,15 +60,20 @@ def enable_button(self, custom_button):
         :type custom_button: QWidget
         """
         custom_button.setEnabled(True)
-        custom_button.setStyleSheet("background-color:#525252; cursor:pointer;" + CadastaStyle.button_style())
+        custom_button.setStyleSheet(
+                "background-color:#525252; cursor:pointer;" +
```
Comment (Gustry) [inline cadasta/gui/tools/cadasta_login.py]: We use single quote for strings : see https://github.com/inasafe/inasafe/wiki/Coding-Standards#strings-and-internationalisation

File: cadasta/gui/tools/cadasta_login.py [hunk 3]
```diff
@@ -60,15 +60,20 @@ def enable_button(self, custom_button):
...
 
     def disable_button(self, custom_button):
         """
         Disable button
         :param custom_button: button that is enabled
         :type custom_button: QWidget
         """
-        custom_button.setStyleSheet("background-color:#A8A8A8;" + CadastaStyle.button_style())
+        custom_button.setStyleSheet(
+                "background-color:#A8A8A8;" + CadastaStyle.button_style()
```
Comment (Gustry) [inline cadasta/gui/tools/cadasta_login.py]: We use single quote for strings : see https://github.com/inasafe/inasafe/wiki/Coding-Standards#strings-and-internationalisation

File: cadasta/gui/tools/cadasta_login.py [hunk 4]
```diff
@@ -83,7 +88,9 @@ def login(self):
 
         if not url or not username or not password:
             self.msg_bar = QgsMessageBar()
-            self.msg_bar.pushWarning("Error", self.tr("URL/Username/password is empty."))
+            self.msg_bar.pushWarning(
+                    "Error", self.tr("URL/Username/password is empty.")
```
Comment (Gustry) [inline cadasta/gui/tools/cadasta_login.py]: Translate the title and use single quote.

File: cadasta/test/test_init.py (hunk 1 above)
Comment (dimasciput) [reply cadasta/test/test_init.py:59]: I actually copy paste from inasafe : https://github.com/inasafe/inasafe/blob/develop/safe/test/test_init.py#L60 , and this is in test case.

Comment (Gustry) [reply cadasta/test/test_init.py:59]: Ok thanks. I didn't see this file was a test sorry.

**Comment by Gustry** [comment]
Looks good to me, thanks !

File: cadasta/gui/tools/cadasta_login.py [hunk 5]
```diff
@@ -87,12 +91,19 @@ def login(self):
 
         if not self.url or not username or not password:
             self.msg_bar = QgsMessageBar()
-            self.msg_bar.pushWarning("Error", self.tr("URL/Username/password is empty."))
+            self.msg_bar.pushWarning(
```
Comment (ismailsunni) [inline cadasta/gui/tools/cadasta_login.py]: It's better to avoid abbreviation in the variable name.

**Review by ismailsunni — APPROVED** [review]
LGTM, just one minor comment

**Pull request merged**